nservicebus/messaging/discard-old-messages.md

```diff
@@ -40,13 +40,13 @@ Because clocks usually are at most a few minutes out of sync this issue only app
 For this reason, it is wise to add the maximum amount of allowed clock offset, called clock drift, to the TTBR value. For example, when using a TimeToBeReceived value of 90 seconds, one should allow for 300 seconds of maximum clock drift, so the TTBR value becomes 90 + 300 = 390 seconds.
 
 
-## Discarding old messages at startup
+## Discarding incoming old messages at startup
 
-In certain situations, it may be required that old messages are not processed after restarting the endpoint. Usually, this functionality is used in development and test environments, but may also be appropriate when messages contain information that gets outdated, e.g. change notifications, readings from sensors in IoT apps, etc.
+In certain situations, it may be required that old messages in the incoming queue should not be processed after restarting the endpoint. Usually, this functionality is used in development and test environments, but may also be appropriate when messages contain information that gets outdated, e.g. change notifications, readings from sensors in IoT apps, etc.
```

Comment:
>it may be required that old messages in the incoming queue should not be processed after restarting the ...

it may be required that existing messages in the incoming queue should not be processed after restarting the ...